manifests/provisioners/plain/resources/deployment.yaml

```diff
@@ -18,7 +18,8 @@ spec:
       serviceAccountName: plain-provisioner-admin
       containers:
         - name: plain-provisioner
-          image: quay.io/operator-framework/plain-provisioner:latest
+          image: quay.io/operator-framework/rukpak:latest
+          command: ["/plain"]
```

Comment:
Do we need to specify an explicit args array and specify the `run` command for the plain provisioner executable?

Reply:
There is no `run` subcommand for the plain provisioner (at least last I checked)